gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/tuples/CSVEdge.java

```diff
@@ -30,35 +30,43 @@ public void setId(String id) {
     f0 = id;
   }
 
-  public String getSourceId() {
+  public String getGradoopIds() {
     return f1;
   }
 
+  public void setGradoopIds(String gradoopIds) {
+    f1 = gradoopIds;
+  }
+
+  public String getSourceId() {
+    return f2;
+  }
+
   public void setSourceId(String sourceId) {
-    f1 = sourceId;
+    f2 = sourceId;
   }
 
   public String getTargetId() {
-    return f2;
+    return f3;
   }
 
   public void setTargetId(String targetId) {
-    f2 = targetId;
+    f3 = targetId;
   }
 
   public String getLabel() {
-    return f3;
+    return f4;
   }
 
   public void setLabel(String label) {
```

Comment:
Please use `@Override` here, since you declared that method in a new `interface`.
(Also not part of this PR, but please add short Javadoc comments for the other methods of this class.)